packages/medulas-react-components/src/components/PageLayout/index.tsx

```diff
@@ -17,24 +17,27 @@ interface Props extends StyleProps {
 
 interface StyleProps {
   readonly color?: 'white' | 'transparent';
+  readonly minHeight?: string;
 }
 
 const useStyles = makeStyles<Theme, StyleProps>({
   root: props => ({
     backgroundColor: props.color,
+    minHeight: props.minHeight,
   }),
 });
 
 const PageLayout = ({
   id,
   children,
-  color = 'transparent',
   title,
   primaryTitle,
   onBack,
+  color = 'transparent',
+  minHeight = '500px',
```

Comment:
You can use: `EXTENSION_HEIGHT` constant.

Reply:
That's why you should use `string | number` because `EXTENSION_HEIGHT` is number type constant.

Reply:
Alright, but `EXTENSION_HEIGHT` is a Sanes' constant, not Medulas', and I don't think there should be imports from Sanes to Medulas. For now I've added the `PAGE_HEIGHT` constant to the PageLayout component to remove the magic number.